Instead of using Google by default, we use pspell to avoid to send each mail to Google...

Index: roundcube/config/main.inc.php.dist
===================================================================
--- roundcube.orig/config/main.inc.php.dist	2012-09-29 11:30:57.895111972 +0200
+++ roundcube/config/main.inc.php.dist	2012-09-29 11:31:46.232078634 +0200
@@ -441,7 +441,7 @@
 
 // Set the spell checking engine. 'googie' is the default. 'pspell' is also available,
 // but requires the Pspell extensions. When using Nox Spell Server, also set 'googie' here.
-$rcmail_config['spellcheck_engine'] = 'googie';
+$rcmail_config['spellcheck_engine'] = 'pspell';
 
 // For a locally installed Nox Spell Server, please specify the URI to call it.
 // Get Nox Spell Server from http://orangoo.com/labs/?page_id=72
